src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -360,44 +360,32 @@
            (throw e)))))))
 
 (defn make-connection-pool
-  "Create a new database connection pool"
-  [{:keys [classname subprotocol subname user username password
-           partition-conn-min partition-conn-max partition-count
-           stats log-statements log-slow-statements statements-cache-size
-           conn-max-age conn-lifetime conn-keep-alive read-only?
-           connection-timeout pool-availability-threshold]
-    :as db}]
-  (let [;; Load the database driver class explicitly, to avoid jar load ordering
-        ;; issues.
-        _ (Class/forName classname)
-        log-slow-statements-duration (pl-time/to-seconds log-slow-statements)
-        config          (doto (new BoneCPConfig)
-                          (.setDefaultAutoCommit false)
-                          (.setLazyInit true)
-                          (.setMinConnectionsPerPartition partition-conn-min)
-                          (.setMaxConnectionsPerPartition partition-conn-max)
```

Comment:
Absolutely, we've been meaning to document the partitioning side of things anyway, now we only want to expose 1 parameter, its easier.